### PR TITLE
Confirm email router

### PR DESCRIPTION
**Pull request by shamilkotta**
### Tasks :-
- [x] Verify email
- [x] Store user 
- [x] Send token

### CODE REVIEW

**Review by Muhammed-Rahif — COMMENTED** [review]
@shamilkotta I checked some requests.. It's looks good to me...👍️
Another change is needed in [`routers/index.js`](https://github.com/jsdcommunity/build_Together_Freelacing-site-for-Developers_BackEnd/blob/5b58a9fdee78a15c0e9752a1cc704c2d65e0e134/routes/index.js#L9) :-

```diff
- router.post("/confirm-email", validateToken, validationResults, createUser)
+ router.post("/confirm-email", validateToken, validationResults, saveUser)
```

**Comment by shamilkotta** [comment]
> @shamilkotta I checked some requests.. It's looks good to me...👍️ Another change is needed in [`routers/index.js`](https://github.com/jsdcommunity/build_Together_Freelacing-site-for-Developers_BackEnd/blob/5b58a9fdee78a15c0e9752a1cc704c2d65e0e134/routes/index.js#L9) :-
> 
> ```diff
> - router.post("/confirm-email", validateToken, validationResults, createUser)
> + router.post("/confirm-email", validateToken, validationResults, saveUser)
> ```
> 
> **Also don't forget to update in other files...**

branch updated check now

File: helpers/index.js [hunk 1]
```diff
@@ -89,4 +103,24 @@ module.exports = {
         }
       });
     }),
+    
+  createUser: (userData)=>
```
Comment (Muhammed-Rahif) [inline helpers/index.js:107]:
```suggestion
  createUser: async (userData)=> 
```

**Review by Muhammed-Rahif — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+config.env
```
Comment (Muhammed-Rahif) [inline .gitignore:2]:
```suggestion
config/config.env
```

Comment (Muhammed-Rahif) [inline .gitignore:2]:
```suggestion
config/config.env
```

Comment (Muhammed-Rahif) [inline .gitignore:2]:
```suggestion
config/config.env
```

**Review by Muhammed-Rahif — CHANGES_REQUESTED** [review]
@shamilkotta

File: .gitignore (hunk 2 above)
Comment (shamilkotta) [reply .gitignore]: Done 
check now

**Review by Muhammed-Rahif — APPROVED** [review]
@shamilkotta now everything looks good.. Let's merge it!

**Pull request merged**